srcpkgs/conky-cli/template

```diff
@@ -1,24 +1,27 @@
 # Template file for 'conky-cli'
 # Keep in sync with 'conky'
 pkgname=conky-cli
-version=1.13.1
+version=1.15.0
 revision=1
+archs="~i686*"
```

Comment:
why?

Reply:
Because it uses long instead if int_32 and so it doesn't pass tests on 32 bit platforms.

Reply:
Is that new in this release? Can we patch it? Do they know about it? Does that also apply to other 32bit archs? Starting to ignore an entire arch is not really nice.